src/pages/Home.js

```diff
@@ -1,23 +1,29 @@
-import React from 'react'
+import React, {useState} from 'react'
 import styled from 'styled-components'
 import Footer from '../components/Footer'
 import Wellcome from '../components/Wellcome'
+import SideDrawerMenu from '../components/sideMenu/SideDrawerMenu'
+import Backdrop from './../components/common/Backdrop'
 
-const Home = () => (
-    <Wrapper>
-        <Wellcome/>
-        <Footer/>
-    </Wrapper>
-)
+const Home = () => {
+    const [isMenuVisible, setMenuVisible] = useState(false);
+
+    return (
+        <Wrapper>
+            <SideDrawerMenu isOpen={isMenuVisible} onCloseMenu={() => setMenuVisible(false)}/>
```

Comment:
as long as you're aware of the rendering costs of passing functions like this it's ok